code/students-performance/src/test/java/ru/unn/agile/StudentsPerformance/Model/PerformanceCalculatorTest.java

```diff
@@ -0,0 +1,219 @@
+package ru.unn.agile.StudentsPerformance.Model;
+
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.Assert.*;
+
+public class PerformanceCalculatorTest {
+
+    private static final double COMPARE_DELTA = 0.001;
```

Comment:
I suppose, delta = 0.1 is enough for students marks

Reply:
Done